build.sbt

```
@@ -1,30 +1,34 @@
 import Tests._
 
-// This gives us a nicer handle  to the root project instead of using the
+// This gives us a nicer handle to the root project instead of using the
 // implicit one
 lazy val chipyardRoot = Project("chipyardRoot", file("."))
 
 lazy val commonSettings = Seq(
   organization := "edu.berkeley.cs",
-  version := "1.0",
+  version := "1.3",
   scalaVersion := "2.12.10",
-  traceLevel := 15,
   test in assembly := {},
   assemblyMergeStrategy in assembly := { _ match {
     case PathList("META-INF", "MANIFEST.MF") => MergeStrategy.discard
     case _ => MergeStrategy.first}},
   scalacOptions ++= Seq("-deprecation","-unchecked","-Xsource:2.11"),
-  libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % "test",
-  libraryDependencies += "org.json4s" %% "json4s-jackson" % "3.6.1",
-  libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value,
-  libraryDependencies += "com.github.scopt" %% "scopt" % "3.7.0",
-  libraryDependencies += "org.scala-lang.modules" % "scala-jline" % "2.12.1",
-  libraryDependencies += "com.typesafe.play" %% "play-json" % "2.6.10",
-  libraryDependencies += "org.typelevel" %% "spire" % "0.16.2",
-  libraryDependencies += "org.scalanlp" %% "breeze" % "1.0",
   addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full),
```

```suggestion
  addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.1" cross CrossVersion.full),
```
The recent bump of scalaVersion to 2.12.12 for mdf causes me to hit:
```
[error] stack trace is suppressed; run last mdf / update for the full output
[error] (mdf / update) sbt.librarymanagement.ResolveException: Error downloading org.scalamacros:paradise_2.12.12:2.1.0
[error]   Not found
[error]   Not found
[error]   not found: /home/centos/.ivy2/local/org.scalamacros/paradise_2.12.12/2.1.0/ivys/ivy.xml
[error]   not found: https://repo1.maven.org/maven2/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: https://oss.sonatype.org/content/repositories/snapshots/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: https://oss.sonatype.org/content/repositories/releases/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
[error]   not found: /home/centos/.m2/repository/org/scalamacros/paradise_2.12.12/2.1.0/paradise_2.12.12-2.1.0.pom
```

Bumping paradise to version 2.1.1 in CY fixes the issue.  We may want to bump to 2.1.1 here as well.

---

Do you know if `paradise` is needed outside of Rocket Chip? Can we just remove that plugin entirely? If not we can bump this.